src/componentes/mounted/lista-nominal/ToolBarMounted.tsx

```
@@ -0,0 +1,87 @@
+}
+
+export const ToolBarMounted = ({
+    updateDate,
+    search,
+    print,
+    inputProps,
+    handleSearchClick
+}: ToolBarMountedProps) => {
+    return (
+        <ListToolBar
+            children={
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)